### PR TITLE
Fix more Rails 6.1 deprecation warnings

### PR DESCRIPTION
**Pull request by jas14**

### CODE REVIEW

File: lib/rpush/client/active_model/apns/device_token_format_validator.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module ActiveModel
       module Apns
         class DeviceTokenFormatValidator < ::ActiveModel::Validator
           def validate(record)
-            return if record.device_token =~ /^[a-z0-9]\w+$/i
+            return if record.device_token =~ /\A[a-z0-9]\w+\z/i
```
Comment (benlangfeld) [inline lib/rpush/client/active_model/apns/device_token_format_validator.rb:7]: Could you provide a reference for this change? Is this just personal preference or does something force us to do it?

**Comment by jas14** [comment]
Looks like this is failing a lot of tests, sorry for the premature PR

File: lib/rpush/client/active_model/apns/device_token_format_validator.rb (hunk 1 above)
Comment (jas14) [reply lib/rpush/client/active_model/apns/device_token_format_validator.rb:7]: Yes, this comes out of [ActiveModel::Validations::FormatValidator](https://github.com/rails/rails/blob/5cfd58bbfb8425ab1931c618d98b649bab059ce6/activemodel/lib/active_model/validations/format.rb#L38-L42).

**Comment by jas14** [comment]
Fixed incorrect use of `ActiveRecord::Errors`

**Review by benlangfeld — APPROVED** [review]

**Pull request merged**